xtask/src/build_ebpf.rs

```diff
@@ -77,9 +77,12 @@ fn build_rust_ebpf(opts: &BuildEbpfOptions) -> anyhow::Result<()> {
 fn get_libbpf_headers<P: AsRef<Path>>(libbpf_dir: P, include_path: P) -> anyhow::Result<()> {
     let dir = include_path.as_ref();
     fs::create_dir_all(dir)?;
+    let mut includedir = std::ffi::OsString::new();
```

Comment:
here and everywhere else - `use` instead of using full paths?

Reply:
done.